.lintrunner.toml

```diff
@@ -232,6 +234,10 @@ include_patterns = [
     'torch/csrc/**/*.cpp',
     'torch/csrc/jit/serialization/*.h',
     'torch/csrc/jit/serialization/*.cpp',
+    'torch/nativert/*.h',
```

Comment:
You need to also modify the `.clang-tidy` file and the header include regex in that file

Reply:
Checked .clang-tidy and seems we already include the entire `torch/` there. https://github.com/pytorch/pytorch/blob/main/.clang-tidy